### DISCUSSION TITLE
Impossibile aggiornare

### DISCUSSION THREAD

**Discussion by baurbr** [indomus/forum#191] [HUB: Home Assistant]
Ciao a tutti,
nella mia installazione di Hassio non è più possibile fare aggiornamenti ne di componenti aggiuntivi ne di HA Core. 
Ho aggiornato l'altro giorno alla versione 2021.4.5 senza problemi.
Ora se provo ad aggiornare mi segnala questo errore
`WARNING (MainThread) [supervisor.jobs] 'AddonManager.update' blocked from execution, no host internet connection`

Qualcuno mi può aiutare a decifrare l'errore e suggerirmi una soluzione? 
Grazie

**Comment by Nik71git** [comment]
a mio avviso la chiave sta tutta nella frase "no host internet connection".... come raggiungi il tuo HA?

**Reply by baurbr** [reply]
aveva confuso anche me quella frase, ma la connessione ad internet c'era e HA lo raggiungevo tranquillamente dall'esterno.
Come ultimo tentativo ho riavviato il raspberry e magicamente tutto è tornato a funzionare

**Reply by Scialla** [reply]
...un riavvio salva la vita :D
